source/operations/external-iam/configure-keycloak-identity-management.rst

```diff
@@ -0,0 +1,259 @@
+      
+      {
+         "id": "954de141-781b-4eaf-81bf-bf3751cdc5f2",
+         "createdTimestamp": 1675866684976,
+         "username": "minio-user-1",
+         "enabled": true,
+         "totp": false,
+         "emailVerified": false,
+         "firstName": "",
+         "lastName": "",
+         "attributes": {
+            "policy": [
+               "readWrite"
+            ]
+         },
+         "disableableCredentialTypes": [],
+         "requiredActions": [],
+         "notBefore": 0,
+         "access": {
+            "manageGroupMembership": true,
+            "view": true,
+            "mapRoles": true,
+            "impersonate": true,
+            "manage": true
+         }
+      }
+
+   MinIO would revoke access for an authenticated user if the returned value has ``enabled: false``, or if the Admin API returned no matching user.
+
+If you do not enable this functionality, the earliest point in time that MinIO could disable access for a disabled or removed user is when the last retrieved authentication token expires.
+
+3) Enable Keycloak Admin Support on MinIO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO supports multiple methods for configuring Keycloak Admin API Support:
+
+- Using a terminal/shell and the :mc-cmd:`mc admin idp openid` command
+- Using environment variables set prior to starting MinIO
+
+.. tab-set::
+
+   .. tab-item:: CLI
+
+      You can use the :mc-cmd:`mc admin idp openid update` command to modify the configuration settings for an existing Keycloak service.
+      You can alternatively include the following configuration settings when setting up Keycloak for the first time.
+      The command takes all supported :ref:`OpenID Configuration Settings <minio-open-id-config-settings>`:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc admin idp openid update ALIAS KEYCLOAK_IDENTIFIER \
+            vendor="keycloak" \
+            keycloak_admin_url="https://keycloak-url:port/admin"
+            keycloak_realm="REALM"
+
+      - Replace ``KEYCLOAK_IDENTIFIER`` with the name of the configured Keycloak IDP.
+        You can use :mc-cmd:`mc admin idp openid list` to view all configured IDP configurations on the MinIO deployment
+        
+      - Specify the Keycloak admin URL to :mc-conf:`keycloak_admin_url <identity_openid.keycloak_admin_url>` configuration setting
+
+      - Specify the Keycloak Realm name to :mc-conf:`keycloak_realm <identity_openid.keycloak_realm>`
+
+   .. tab-item:: Environment Variables
+
+      Set the following :ref:`environment variables <minio-server-envvar-external-identity-management-openid>` in the appropriate configuration location, such as ``/etc/default/minio``.
+
+      The following example code sets the minimum required environment variables related to enabling the Keycloak Admin API for an existing Keycloak configuration.
+      Replace the suffix ``_KEYCLOAK`` with the unique identifier for the target Keycloak configuration.
```

Comment:
I'm not clear what this sentence means.
So `MINIO_IDENTITY_OPENID_VENDOR_KEYCLOAK="keycloak"` might become
`MINIO_IDENTITY_OPENID_VENDOR_MYKEYTARGET="keycloak"`?

Reply:
Basically yes. Not sure how to approach this to be honest. Open to suggestions!

Reply:
So I think my suggestion would be something like

```
         MINIO_IDENTITY_OPENID_VENDOR_{SUFFIX}="keycloak"
         MINIO_IDENTITY_OPENID_KEYCLOAK_ADMIN_URL_{SUFFIX}="https://keycloak-url:port/admin"
```

Then say, 
> Replace `{SUFFIX}` with a unique identifier for your keycloak IDP. For example, `MINIO_IDENTITY_OPENID_VENDOR_KEYCLOAK`. 
If you do not expect to have more than one OpenID vendor, you can leave the suffix off entirely and use `MINIO_IDENTITY_OPENID_VENDOR`.